src/services/zapper.js

```diff
@@ -61,6 +61,10 @@ type ZapperProtocolBalances = {
   }
```

Comment:
❗ important
Commenting here, because I cannot comment in the non-related code in this file:

Above when defining ZapperAsset we are using `balance: number` to read asset balances. This will potentially result in subtle rounding errors when user would try to use send max later in the app.  According to zapper swagger api they also include `"balanceRaw": "string",` field which seems to be exact value as string.